bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/template/TemplateParametersEditor.scala

```diff
@@ -257,20 +256,25 @@ class TemplateParametersEditor(shells: java.util.List[ISPTemplateParameters]) ex
 
       val JNoneLong: JOption[java.lang.Long] = JNone.instance[java.lang.Long]
 
+      // These are defined in object SPTarget, but private.  Can we open them
+      // up and/or provide get/set for RightAscension/Declination on SPTarget?
+      val ra:  Target @?> RightAscension = Target.coords >=> Coordinates.ra.partial
```

Comment:
I guess I wanted to see what @tpolecat had to say first since he's actively working on `SPTarget` and friends.  I think perhaps get/set pairs for `RightAscension` and `Declination` might make sense?